### PR TITLE
monodis: add page

### PR DESCRIPTION
**Pull request by sbrl**
As with #2557, this was very difficult to keep it down to just 8 examples. For both, I've tried to pick the most common options and present them in a form that's logic to one's workflow. Particularly with `csc` though it's unlikely that you'll be interfacing with it directly (since `msbuild` does that for you!) unless you've got a really odd setup.

----

- [ ] The page (if new), does not already exist in the repo.

- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [ ] The page has 8 or fewer examples.

- [ ] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [ ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/monodis.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+> The Mono Common Intermediate Language (CIL) disassembler.
+
+- Disassemble an assembly to textual CIL:
+
+`monodis {{path/to/assembly.exe}}`
+
+- Save the output to a file:
+
+`monodis --output={{path/to/output.il}}`
```
Comment (agnivade) [inline pages/common/monodis.md]: path/to/assembly.exe missing ?

File: pages/common/monodis.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# monodis
+
+> The Mono Common Intermediate Language (CIL) disassembler.
+
+- Disassemble an assembly to textual CIL:
```
Comment (agnivade) [inline pages/common/monodis.md]: Should this be "Disassembly a binary" ? And same below where `exe`s are used.

File: pages/common/monodis.md (hunk 1 above)
Comment (sbrl) [reply pages/common/monodis.md]: Good catch!

File: pages/common/monodis.md (hunk 2 above)
Comment (sbrl) [reply pages/common/monodis.md]: Indeed they are binaries, but `assembly` is the technical .NET term for a _Common Intermediate Language_ binary.

**Comment by sbrl** [comment]
I think you're broken, @stale bot.

**Pull request merged**